sources/editor/Stride.Assets.Presentation/Templates/Assets/Scripts/Events/EventListener.cs

```diff
@@ -3,39 +3,37 @@
 using Stride.Engine;
 using Stride.Engine.Events;
 
-namespace ##Namespace##
+namespace ##Namespace##;
+
+/// <summary>
+/// Script which listens to an event, and then catches it and performs some game logic based on the it.
+/// </summary>
+public class ##Scriptname## : AsyncScript
 {
+    // Hint. You also need the EventBroadcaster script. Change the namespace accordingly.
+    private readonly EventReceiver<string> listener = new (EventBroadcaster.EventKey);
```

Comment:
nit: I think in this style, there is no space between `new` and `(`.

Reply:
This spacing was intentional, but sure I can change that

Reply:
Done